### PR TITLE
pyjsparser 2.7.1

### PR DESCRIPTION
**Pull request by psteyer**
adding abs yaml file with organization and channel for testing purposes in order to build Lark

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
we most likely want to remove the `.DS_Store` file

**Review by varlackc — COMMENTED** [review]
We are currently trying to avoid building `noarch` packages, we might have to modify the package.

We might want to remove the quotes from the `home url` that will make it easier to copy and paste the links for review.

**Comment by skupr-anaconda** [comment]
There are missing `setuptools` and `wheel` in `host`. 
Also, it would be nice to add `pip check`

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,27 +10,31 @@ source:
...
     - python
 
 test:
   imports:
     - pyjsparser
+  command:
+    - pip check
 
 about:
   home: "https://github.com/PiotrDabkowski/pyjsparser"
```
Comment (varlackc) [inline recipe/meta.yaml]: We might want to remote the quotation marks around the `home url`:

https://github.com/AnacondaRecipes/pyjsparser-feedstock/blob/b8d314d3cdf75d40f3c516b724b39b9f6f5e9894/recipe/meta.yaml#L33

Comment (psteyer) [reply recipe/meta.yaml]: removed quotations

**Review by skupr-anaconda — APPROVED** [review]

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**